### ISSUE TITLE
phpmyadmin does not work after moving the home directory

### ISSUE THREAD

**Issue by Whip** [virtualmin/virtualmin-gpl#362]
I used the Change Domain Name to move the home directory but phpmyadmin stopped working after doing so. I get the following error
```
Error during session start; please check your PHP and/or webserver log file and configure your PHP installation properly. Also ensure that cookies are enabled in your browser.
session_start(): open(SESSION_FILE, O_RDWR) failed: No such file or directory (2)
session_start(): Failed to read session data: files (path: /home/oldlocation/tmp)
```
I asked this question before on the [forums](https://forum.virtualmin.com/t/phpmyadmin-not-working-after-domain-rename/108826/8) but there was no solution.

**Comment by jcameron** [comment]
@iliarostovtsev is this something that can be handled by your code that updates scripts if the MySQL login changes?

**Comment by iliarostovtsev** [comment]
Yes, it _could_ but we don't set **`$cfg['SessionSavePath']`** and I don't think we should touch it.

@Whip If you set it manually, do you expect Virtualmin to update it?

**Comment by iliarostovtsev** [comment]
Sorry, are you asking for help solving this problem or request a feature for Virtualmin to do it automatically?

> session_start(): Failed to read session data: files (path: /home/oldlocation/tmp)

Simply check _`/public_html/phpmyadmin/config.inc.php`_ and update the path.

Also, you could quickly find for occurrences of an old path _`oldlocation`_ by using File Manager search or using the following command:

```
grep -R oldlocation /home/newlocation
```

.. and fix it manually.

**Comment by Whip** [comment]
I tried searching for the oldlocation but got no results. I've also tried clearing the `tmp` folder of phpmyadmin. Not sure where this is coming from.

Were you able to replicate the problem?

**Comment by iliaross** [comment]
Reopened, as this was closed by an automated cleanup run.

⚠️ If the issue is resolved, please take a moment to close it to help keep the project organized.

**Comment by Whip** [comment]
I moved a site a few months back. I think there was no issue with phpmyadmin that time. We can reopen later if the issue crops up.
